### PR TITLE
BEIS ingress v3 - electricity

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] Update

**What category or categories does it affect?**

_Electricity_

**What?**

_Adding BEIS electricity category to the OEFDB._

**Why?**

_To have the complete set of BEIS data points in the OEFDB._

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -393,68 +393,68 @@ Energy,Electricity,electricity-energy_source_solar_thermal,Electricity generated
...
+Energy,Electricity,electricity-energy_source_grid_mix,Electricity generation – WTT,kWh,not-supplied,0.1755705744,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,AT,,44640,Emission intensity of WTT electricity generation Austin. Well-to-Tank (WTT) conversion factors for UK and overseas electricity should be used to report the Scope 3 emissions of extraction and refining and transportation of primary fuels before their use in the generation of electricity. It should be noted that electricity generation and transmission and distribution losses have separate WTT emissions assigned to them. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:396]: looks like the data columns has decoded itself!

Comment (patstirling) [reply OpenEmissionFactorsDB.csv:396]:
```suggestion
Energy,Electricity,electricity-energy_source_grid_mix,Electricity generation – WTT,kWh,not-supplied,0.1755705744,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,AT,,44640,Emission intensity of WTT electricity generation Austria. Well-to-Tank (WTT) conversion factors for UK and overseas electricity should be used to report the Scope 3 emissions of extraction and refining and transportation of primary fuels before their use in the generation of electricity. It should be noted that electricity generation and transmission and distribution losses have separate WTT emissions assigned to them. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -393,68 +393,68 @@ Energy,Electricity,electricity-energy_source_solar_thermal,Electricity generated
...
+Energy,Electricity,electricity-energy_source_grid_mix,Electricity generation – WTT,kWh,not-supplied,0.04868811525,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,AU,,44640,Emission intensity of WTT electricity generation Australia. Well-to-Tank (WTT) conversion factors for UK and overseas electricity should be used to report the Scope 3 emissions of extraction and refining and transportation of primary fuels before their use in the generation of electricity. It should be noted that electricity generation and transmission and distribution losses have separate WTT emissions assigned to them. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:397]: Australia and Austria seem to be flipped?

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1123,69 +1123,70 @@ Energy,Electricity,electricity-energy_source_grid_mix_residual_mix_market_based,
...
+Energy,Electricity,electricity-energy_source_grid_mix_losses,Transmission & distribution – WTT,kWh,not-supplied,0.01358799308,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,well_to_tank,BEIS,2021,2021,,EURASIA,,44640,Emission intensity of WTT transmission & distribution Non-OECD Europe and Eurasia (average). Well-to-Tank (WTT) conversion factors for UK and overseas electricity should be used to report the Scope 3 emissions of extraction and refining and transportation of primary fuels before their use in the generation of electricity. It should be noted that electricity generation and transmission and distribution losses have separate WTT emissions assigned to them. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:1188]: Eurasia region is in the custom regions already?

**Review by AnastasiaLobanova — APPROVED** [review]

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:1188]: All looks good, just the date_accessed column is off and probably we need some custom regions for the API?

**Review by patstirling — APPROVED** [review]
Approved so that you can get on with releasing this, but the dates are in format DD/M/YYYY rather than DD/MM/YYYY in the date accessed column. I've also spotted a bunch of strange dates in there, from a copy and paste error I think from the iterative pattern. Suggest this column get a review and standardisation in a follow-up PR.

**Pull request merged**